### PR TITLE
Hidden Long Breadcrumbs

### PR DESCRIPTION
**Pull request by martypowell**
When breadcrumbs get really long, we won't to hide some of them. We will show a dropdown list with the additional items so that the user still has access each link.

For now on smaller screens, the breadcrumbs will be hidden.

### CODE REVIEW

File: js/breadcrumbs.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+			return `<li>${$clonedBreadCrumb[0].outerHTML}</li>`;
+		}).join('');
+
+		return listItems.length ? `<ul class="hidden-breadcrumb-list">${listItems}</ul>` : '';
+	};
+
+	const buildBreadCrumbTrigger = (breadCrumbTrigger) => `<span class="${breadCrumbTrigger}"><i class="fa fa-circle" aria-hidden="true"></i><i class="fa fa-circle" aria-hidden="true"></i><i class="fa fa-circle" aria-hidden="true"></i></span>`;
+
+	const isEmpty = ($elm) => !$.trim($elm.html().replace(/(?:\r\n|\r|\n|\s|&nbsp;)/g, ''));
```
Comment (mxsnyder) [inline js/breadcrumbs.js]: Eventually, this can be moved to a utility class.

File: js/breadcrumbs.js [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+			$elm.show();
+		} else {
+			$elm.hide();
+		}
+	};
+
+	const init = (breadcrumbThreshold) => {
+		const breadcrumbThresholdLimit = breadcrumbThreshold || 3;
+
+		$(function onDocumentReady() {
```
Comment (mxsnyder) [inline js/breadcrumbs.js]: Since you need to run init anyway, do you need this to run on document.ready?

File: js/breadcrumbs.js (hunk 1 above)
Comment (martypowell) [reply js/breadcrumbs.js]: @mxsnyder good idea, tracked here, https://github.com/baltimorecounty/BCPL-assets/issues/184

**Comment by martypowell** [comment]
@mxsnyder fyi, wrapped document .ready around the inti call rather than in the function

**Pull request merged**